pipeline/clean/clean-mono.sh

```diff
@@ -42,19 +42,21 @@ else
         | pigz >"${output_prefix}.${lang}.monofix.gz"
 fi
 
+# disabled bue to errors in langid, need to debug sometime
```

Comment:
should we just add a config setting to disable it? I don't think we had problems with it for our training